Dockerfile

```diff
@@ -130,10 +130,10 @@ ARG PYTHON_VERSION=3.10.8
 ARG PYENV_GIT_TAG=v2.3.7
 
 ENV PYENV_ROOT=/opt/python
-ENV PATH=$PATH:${PYENV_ROOT}/shims:${PYENV_ROOT}/bin
+ENV PATH=$PATH:$PYENV_ROOT/shims:$PYENV_ROOT/bin
```

Comment:
I haven't done much shell programming in recent years, but I remember that it was considered a best practice to always use braces for consistency and readability, and to reduce the possibility of mistakes. Sometimes they might not be required, but to the (unexperienced) user it might still be ambiguous how to read an expression.

Reply:
I looked up the related Shellcheck rule which has an example and rationale for why it might be good to keep the braces: https://www.shellcheck.net/wiki/SC2250

Reply:
> I looked up the related Shellcheck rule which has an example and rationale for why it might be good to keep the braces: https://www.shellcheck.net/wiki/SC2250

Thanks for the reference, but even Shellcheck says

> This is an [optional](https://www.shellcheck.net/wiki/optional) suggestion. It must be explicitly enabled with a [directive](https://www.shellcheck.net/wiki/directive)

So it's *disabled* by default. And I actually disagree regarding the readability: To mee fewer special chars are less distracting than having more of them. Also, syntax-highlighting guards even the inexperienced user against escaping mistakes.

Reply:
Yes, it's optional, but it's still a suggestion. But I'm fine with removing them, I just wanted to make sure it's considered.

Reply:
As a side note, Kotlin inspection has exactly the opposite hint: It suggests to remove superfluous curly braces from string interpolations like `"${it}"`. I prefer the Kotlin style :wink: